demisto_sdk/commands/common/tests/update_id_set_test.py

```diff
@@ -680,7 +686,7 @@ def test_get_script_data_script_executions(repo, code):
             'commonfields': {
                 'id': 'DummyScript'
             }})
-        res, _ = process_script(script.path, packs={'DummyPack': {}}, marketplace=MarketplaceVersions.XSOAR.value, print_logs=False)
+        res, _ = process_script(script.yml.path, packs={'DummyPack': {}}, marketplace=MarketplaceVersions.XSOAR.value, print_logs=False)
```

Comment:
How come this worked before?

Reply:
Reverting this.
It was supposed to be a test fix, but I fixed it in another way and forgot to revert.